### ISSUE TITLE
Different wannier calculation results from QE

### ISSUE THREAD

**Issue by shiruosong** [deepmodeling/abacus-develop#2863]
### Describe the bug

After generating the wannier.amn/wannier.mmn/wannier.eig file using ABACUS, I calculated the Maximum Local Wannier Function (MLWF). But the Wannier calculation does not converge. The spread of each Wannier center is quite large, even larger than 10. That's very bad. Even for the provided examples (interface_wannier90), MLWC calculated by ABACUS is not convergent and the spread is too large.

Then I do the same calculations using QE with the same pseudopotentials and the same structure. Now, it converges pretty well, with a spread of ony about 1~2 per wannier center.

In summary, there might be some problems with the interface to the wannier90, and the MLWC we get from abacus is not correct.
